components/user/UserTestimonials.js

```diff
@@ -31,11 +32,13 @@ export default function UserTestimonials({ users, data, BASE_URL }) {
             key={key}
           >
             <div className="flex-none p-6">
-              <img
-                src={`https://github.com/${testimonial.username}.png`}
-                alt={testimonial.username}
-                className="h-20 w-20 rounded-full bg-gray-100"
-              />
+              <a href={`https://github.com/${testimonial.username}`} target="_blank" rel="noreferrer">
```

Comment:
I don't think we need to craft the url here as it is already used https://github.com/EddieHubCommunity/LinkFree/pull/2503/files#diff-bb3dd05a9b343b13a69d788e6e57a1d754e4d5bde52886480e08677f6c26ca9eR45

this is crafted here https://github.com/EddieHubCommunity/LinkFree/pull/2503/files#diff-bb3dd05a9b343b13a69d788e6e57a1d754e4d5bde52886480e08677f6c26ca9eR12 and here https://github.com/EddieHubCommunity/LinkFree/pull/2503/files#diff-bb3dd05a9b343b13a69d788e6e57a1d754e4d5bde52886480e08677f6c26ca9eR19